README.md

```diff
@@ -1,5 +1,5 @@
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/kokkos/kokkos-kernels/badge)](https://securityscorecards.dev/viewer/?uri=github.com/kokkos/kokkos-kernels)
-[![Generic badge](https://readthedocs.org/projects/kokkos-kernels/badge/?version=latest)](https://kokkos-kernels.readthedocs.io/en/latest/)
+[![kokkos-kernels/docs](https://github.com/kokkos/kokkos-kernels/actions/workflows/docs.yml/badge.svg)](https://github.com/kokkos/kokkos-kernels/actions/workflows/docs.yml?query=branch%3Adevelop)
```

Comment:
What's your intent here?
Telling the reader whether the documentation workflow is broken and show them where it is tested?
Or should that be a shortcut that redirects to the actual documentation

Reply:
That's a fair question, here the update was just a fix to not report status on the old deprecated API documentation.
I guess I could have just deleted the old button... it might be more appropriate to link to the documentation itself rather than the action that built it last, that would be more appropriate to do that on our regular meeting agendas...

Reply:
The previous link was redirecting to the actual doc.  This what prompted my question.